install.php

```diff
@@ -1676,6 +1731,10 @@ function BuildFileList(){
 			<div><input type="text" size="70" defaultvalue="',$config->defaults["LDAPContactAdmin"],'" name="LDAPContactAdmin" value="',$config->ParameterArray["LDAPContactAdmin"],'"></div>
 		</div>
 		<div>
+			<div><label for="LDAPBulkOperations">',__("Bulk Operations"),'</label></div>
```

Comment:
When the BulkOperations priv was introduced, it was missed out from the installer.